import "bootes/dances/proto/Stage.proto";
package pb;

message TeachRecord {
  required int32 time  = 1;
  required Accel accel = 10;
  required Gyro  gyro  = 11;
}

message TeachSequence {
  required string      uuid    = 1;
  required bool        succeed = 2;
  repeated TeachRecord records = 3;
}

message TeachClear {
  required string      uuid    = 1;
}

message TeachCommand {
  required string        uuid     = 1;
  optional TeachSequence sequence = 10;
  optional TeachClear    clear    = 11;
}
message TeachLog {
  repeated TeachCommand commands = 1;
}
